### ISSUE TITLE
Error: Cannot find module 'async'

### ISSUE THREAD

**Issue by harrisonhoward** [alfg/overwatch-api#91]
Making a Discord Bot using this module.

This is the snippet of the code that's causing the error
```js
    /**
     * Validates a provided Overwatch Account
     * @param {String} platform The users platform
     * @param {String} name The users name
     * @returns {Boolean}
    */
    overwatchValidate(platform, name) {
        return new Promise((resolve, reject) => {
            const overwatch = require("overwatch-api");
            const validatePlatforms = ["pc", "xbl", "psn", "switch"];

            if (!validatePlatforms.includes(platform)) {
                reject(new Error("Invalid Platform Provided"));
            }

            overwatch.getProfile(platform, "global", name, err => {
                if (err) {
                    reject(err);
                } else {
                    resolve(true);
                }
            });
        });
    }
```

This is the error
```
Error: Cannot find module 'async'
Require stack:
- C:\Users\----\Documents\----\ErisBot\node_modules\overwatch-api\lib\parser\profile.js
- C:\Users\----\Documents\----\ErisBot\node_modules\overwatch-api\lib\parser\index.js
- C:\Users\----d\Documents\----\ErisBot\node_modules\overwatch-api\lib\index.js
- C:\Users\----d\Documents\----\ErisBot\library\functions\account.js
- C:\Users\----\Documents\----\ErisBot\library\functions\index.js
- C:\Users\----\Documents\----\ErisBot\index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:982:15)
    at Function.Module._load (internal/modules/cjs/loader.js:864:27)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (C:\Users\----\Documents\----\ErisBot\node_modules\overwatch-api\lib\parser\profile.js:10:37)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
```

**Comment by harrisonhoward** [comment]
I fixed it by just doing `npm instal async` rather weird that it doesn't get installed with the module itself.

**Comment by alfg** [comment]
Ah, good catch. I think I added the `async` module to `devDependencies` by mistake, instead of just `dependencies` for the  module:
https://github.com/alfg/overwatch-api/blob/master/api/package.json#L22

I'll look into fixing this soon.

**Comment by alfg** [comment]
This is fixed in latest [0.10.3](https://www.npmjs.com/package/overwatch-api/v/0.10.3). Thanks!
